cmd/runtimetest/main.go

```diff
@@ -642,6 +644,12 @@ func validate(context *cli.Context) error {
 			err := v.test(spec)
 			t.Ok(err == nil, v.description)
 			if err != nil {
+				if logLevel >= logrus.ErrorLevel {
+					// Skip 'SHOULD', 'MAY' ...
```

Comment:
I'll update with all your good suggestions tomorrow. It is a little bit late here.